test/end-to-end/pages/screeners/willoriginal.js

```diff
@@ -1,17 +1,22 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalEn = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalCy = require('app/resources/cy/translation/screeners/willoriginal');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function(answer) {
+module.exports = async function(language ='en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
+    const willOriginalContent = language === 'en' ? willOriginalEn : willOriginalCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/willoriginal');
-    await I.waitForText(content.question);
+    if (language === 'en') {
+        await I.waitForText(willOriginalContent.question);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages